packages/core/src/lib/unique-id.ts

```diff
@@ -0,0 +1,41 @@
+import { setContext, getContext } from 'svelte';
+import { readable, type Readable, get } from 'svelte/store';
+
+type GetCount = () => number;
+type UniqueIdStore = Readable<GetCount>;
+
+const CONTEXT_KEY = 'prime-core-uid';
```

Comment:
A `Symbol` is ever so slightly safer, since it cannot collide

```suggestion
const CONTEXT_KEY = Symbol('prime-core-uid');
```